src/ERC4337Account.sol

```diff
@@ -24,4 +24,15 @@ abstract contract ERC4337Account is IERC4337Account {
     }
 
     function ENTRY_POINT() public view virtual returns (address);
+
+    /// @notice Packs the entry point into a uint256.
+    /// @dev Set the most significant bit to 1 to indicate that the entrypoint has been set by the user.
+    function _packEntryPoint(address entryPoint) internal pure returns (uint256) {
```

Comment:
put in library?

entrypoint.pack()

entrypoint.isSet()

Reply:
or maybe entrypoint.isOverridden() is more intuitive here? tells what the "set" flag does

Reply:
just added!